### ISSUE TITLE
Taint and Tolerations note on special cases doers not render properly

### ISSUE THREAD

**Issue by briandealwis** [kubernetes/website#21204]
**This is a Bug Report**

**Problem:**

The [**Note** on special cases in the _Taint and Tolerations_](https://github.com/kubernetes/website/blame/master/content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md#L76-L96) document is not rendered correctly:

![Screen Shot 2020-05-26 at 9 56 44 AM](https://user-images.githubusercontent.com/202851/82909389-3639d980-9f37-11ea-900e-8fea2811cfd9.png)

**Proposed Solution:**

In-lining this special cases text by removing the `{{ <note> }}` call-out seems reasonable?

**Page to Update:**
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

**Comment by sftim** [comment]
This is related to tracking issue #20335.

The long term fix is probably along the lines of https://github.com/kubernetes/website/issues/20780

For a work-around, how about rewording the note so that it remains called out but does not use a bulleted list?

/kind bug
/language en
